### ISSUE TITLE
Problematic file name in lidapy installation

### ISSUE THREAD

**Issue by sfstrain** [CognitiveComputingResearchGroup/lidapy-framework#41]
The file "./scripts/setup.sh" is a file that setups up lidapy and should be run only ONCE.

It has the same name as files found in the "devel" folder of a ros workspace, which should be run any time one wants to configure their ros environment in a certain way. 

The identical name seems unnecessary, and has the potential for confusing a user into believing that the /scripts/setup.sh file could be safely run more than once--this may not be the case!

It is suggested to change the filename to "setup_lidapy.sh" or something similar to avoid this confusion.

**Comment by skugele** [comment]
Renaming this script to ```build_workspace.sh``` to avoid naming similarities with ROS env scripts and make the intent of this script more clear.

**Comment by skugele** [comment]
The changes have been made and merged into the master branch.  Assigning to the originator ( @sfstrain ) to approve the changes and close the issue.

**Comment by sfstrain** [comment]
The changes addressed the issue, so I'm closing it.
